test/torchaudio_unittest/transforms/transforms_test.py

```diff
@@ -56,7 +56,7 @@ def test_AmplitudeToDB(self):
 
     def test_melscale_load_save(self):
         specgram = torch.ones(1, 1000, 100)
-        melscale_transform = transforms.MelScale()
+        melscale_transform = transforms.MelScale(n_stft=1000)
```

Comment:
Instead of setting `n_stft`, can we change the shape of `specgram` to `1, 201, 100`?
This is more of a personal preference but `1000` is somehow conspicuous.